sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseNetworkHelper.java

```diff
@@ -35,20 +36,20 @@ HttpRequest buildPingRequest(
         String instanceName) {
 
         HttpRequest request = buildRequest(currentDate, address);
-        request.setHeader(QPS_ROLE_NAME_HEADER, roleName);
-        request.setHeader(QPS_MACHINE_NAME_HEADER, machineName);
-        request.setHeader(QPS_STREAM_ID_HEADER, quickPulseId);
-        request.setHeader(QPS_INSTANCE_NAME_HEADER, instanceName);
+        request.setHeader(HttpHeaderName.fromString(QPS_ROLE_NAME_HEADER), roleName);
+        request.setHeader(HttpHeaderName.fromString(QPS_MACHINE_NAME_HEADER), machineName);
+        request.setHeader(HttpHeaderName.fromString(QPS_STREAM_ID_HEADER), quickPulseId);
+        request.setHeader(HttpHeaderName.fromString(QPS_INSTANCE_NAME_HEADER), instanceName);
         request.setHeader(
-            QPS_INVARIANT_VERSION_HEADER, Integer.toString(QuickPulse.QP_INVARIANT_VERSION));
+            HttpHeaderName.fromString(QPS_INVARIANT_VERSION_HEADER), Integer.toString(QuickPulse.QP_INVARIANT_VERSION));
```

Comment:
i didn't follow how these changes are related to OpenTelemetry update?

Reply:
deprecated warning